mobile/lib/providers/album/album.provider.dart

```diff
@@ -94,3 +136,31 @@ final albumRenderlistProvider =
   }
   return const Stream.empty();
 });
+
+class LocalAlbumsNotifier extends StateNotifier<List<Album>> {
+  LocalAlbumsNotifier(this.db) : super([]) {
+    final query = db.albums.filter().not().remoteIdIsNotNull();
```

Comment:
Can we please not use a double negation?

Reply:
alright, I was just hinting at doing `db.albums.where().remoteIdIsNull()` instead :)